src/sdk.js

```diff
@@ -406,6 +406,27 @@ const endpointDefinitions = [
     method: 'get',
     interceptors: [new TransitResponse()],
   },
+  {
+    apiName: 'api',
+    path: 'stripe_account/create',
+    internal: false,
+    method: 'post',
+    interceptors: [new TransitResponse(), new TransitRequest()],
+  },
+  {
+    apiName: 'api',
+    path: 'stripe_account/update',
+    internal: false,
+    method: 'post',
+    interceptors: [new TransitResponse(), new TransitRequest()],
+  },
+  {
+    apiName: 'api',
+    path: '/stripe_persons/create',
```

Comment:
Other endpoint descriptions don't have the leading `/`. I'm not sure if having it here breaks something, but good to remove for consistency at least.

Reply:
D'oh.